v1/serviceEntity/manifest-wrapper.yaml

```diff
@@ -0,0 +1,12 @@
+title: ManifestsWrapper
+type: object
+required:
+- sources
+properties:
+  primary:
```

Comment:
What does primary mean in manifests?

Reply:
in helm deployment we have primary manifest concept.
There user is allowed to give multiple manifest in helm deployment.